File: module_course/src/main/java/com/yhklsdf/module_course/CourseActivity.java
package com.yhklsdf.module_course;

import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class CourseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_course);
        FragmentManager manager = getSupportFragmentManager();
        FragmentTransaction transaction = manager.beginTransaction();
        transaction.replace(R.id.framelayout, new CourseFragment());
        transaction.commit();
    }
}

File: module_course/src/main/java/com/yhklsdf/module_course/CourseActivity.kt
package com.yhklsdf.module_course

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.alibaba.android.arouter.facade.annotation.Route

@Route(path = "/course/main")
class CourseActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_course)
        supportFragmentManager.beginTransaction().replace(R.id.container, CourseFragment()).commit()
    }
}
